src/components/calculator/PrevalenceControls.tsx

```diff
@@ -306,59 +322,48 @@ export const PrevalenceControls: React.FunctionComponent<{
           />
         </div>
       )}
-      {isManualEntryCurrently ? (
-        <span>
-          <button
-            id="switchBetweenManualDataAndLocationSelection"
-            className="btn btn-link text-muted"
-            onClick={handleSelectLocationButtonOnClick}
-          >
-            {t('calculator.switch_button.select_location')}
-          </button>
-        </span>
-      ) : (
-        <span>
-          <button
-            id="switchBetweenManualDataAndLocationSelection"
-            className="btn btn-link text-muted"
-            onClick={handleEnterDataButtonOnClick}
-          >
+      <span>
+        <ToggleButton
+          id="switchBetweenManualDataAndLocationSelection"
+          name={t('calculator.switch_button.select_location')}
+          type="checkbox"
+          checked={isManualEntryCurrently}
+          value="1"
+          variant="link"
+          className="text-muted"
+          onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+            setManualPrevalenceData(e.currentTarget.checked)
+          }}
+        >
+          <span id="switchBetweenManualDataAndLocationSelectionText">
             {t('calculator.switch_button.enter_data_manually')}
-          </button>
-        </span>
-      )}
-      <ControlledExpandable
-        id="prevalence-details"
-        header={t('calculator.prevalence.details_header')}
-        headerClassName={isManualEntryCurrently ? 'd-none' : ''}
-        open={detailsOpen}
-        setter={setDetailsOpen}
-      >
-        <PrevalenceResult data={data} />
-
-        <PrevalenceField
-          id="reported-cases"
-          label={t('calculator.prevalence.last_week_cases')}
-          value={data.casesPastWeek || 0}
-          setter={(value) =>
-            setter({ ...data, casesPastWeek: parseInt(value || '') })
-          }
-          inputType="number"
-          isEditable={isManualEntryCurrently}
-          className="hide-number-buttons"
-        />
-        <PrevalenceField
-          id="population"
-          label={t('calculator.prevalence.population')}
-          value={data.population}
-          setter={(value) => setter({ ...data, population: value })}
-          inputType="text"
-          isEditable={isManualEntryCurrently}
-          className="hide-number-buttons"
-        />
-        {locationSet && data.casesIncreasingPercentage === 0 ? (
-          <div>{t('calculator.prevalence.cases_stable_or_decreasing')}</div>
-        ) : (
+          </span>
+        </ToggleButton>
+      </span>
+      {isManualEntryCurrently ? (
```

Comment:
The two chunks that get swapped between here seem like a good candidate for refactoring into their own components. Possibly put them in new files, components/calculator/prevalence/LocationPrevalenceDetails.tsx and components/calculator/prevalence/ManualPrevalenceDetails.tsx

Could be deferred to a later PR.

Reply:
done